Subject: [PATCH] JIRA-1234: updated project structure
---
Index: src/Data Types and Variables/com/Boolean/Boolean.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Data Types and Variables/com/Boolean/Boolean.java b/src/Data Types and Variables/com/Boolean/Boolean.java
--- a/src/Data Types and Variables/com/Boolean/Boolean.java	(revision c17580cd60f84953154de2fb1c6539148699371b)
+++ b/src/Data Types and Variables/com/Boolean/Boolean.java	(date 1748095522292)
@@ -20,5 +20,8 @@
 		// Change the value of boolean variable
 		flag1 = false;
 		System.out.println("First Flag: " + flag1);
+
+		boolean flag2 = false;
+		System.out.println("Second Flag" + flag2);
 	}
 }
Index: src/Data Types and Variables/com/Charachter/Charachter.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Data Types and Variables/com/Charachter/Charachter.java b/src/Data Types and Variables/com/Charachter/Charachter.java
--- a/src/Data Types and Variables/com/Charachter/Charachter.java	(revision c17580cd60f84953154de2fb1c6539148699371b)
+++ b/src/Data Types and Variables/com/Charachter/Charachter.java	(date 1748095584692)
@@ -17,5 +17,9 @@
 
 		// Print the value of char variable
 		System.out.println("Charachter is: " + a);
+
+		char c;
+
+		c = 'd';
 	}
 }
